src/core/js/views/adaptView.js

```diff
@@ -38,6 +39,25 @@ define([
             return this;
         },
 
+        setupOnScreenHandler: function() {
+            var onscreen = this.model.get('_onScreen');
+
+            if (onscreen && onscreen._isEnabled) {
```

Comment:
Would you mind terribly reversing the logic here so we can sink the indentation?
```js
if (!onscreen || !onscreen._isEnabled) return;
```